tools/samtool_filter2/samtool_filter2.xml

```diff
@@ -59,7 +59,9 @@ samtools view $possibly_select_inverse '$output1' $header
   #end if
   $input
   #if str($regions).strip() and $input1.is_of_type('bam')
-    '$regions'
+    #for region in str($regions).split(' '):
```

Comment:
Maybe it would be better to split on any whitespace character, i.e. use `.split()`